### DISCUSSION TITLE
Allow including more folders to watch for refreshing the app when code is changed

### DISCUSSION THREAD

**Discussion by abishekrsrikaanth** [NativePHP/laravel#146] [Ideas]
Currently only changes to `NativePHPServiceProvider` is being watched to refresh the app when a change happens, it would be great if there was a way to include more folders to watch for changes.

**Comment by mpociot** [comment]
The watcher is only used for restarting your Electron app. This is basically only necessary for anything running during app startup (like menu, menubar, root window).
If you want hot-reloading for your frontend, you should use Vite instead.
Just run `npm run dev` separately for this to work.

**Comment by simonhamp** [comment]
You'll [soon](https://github.com/NativePHP/electron/pull/119) be able to add a Composer script that will more easily let you run these side by side from a single command
